schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE animals (
+   id integer PRIMARY KEY,
+   name TEXT,
```

Comment:
- [ Optional ] Kindly I recommend you to change the data type of` name` into : `varchar(255`) instead of `TEXT`,